tests/unit/test_service.py

```diff
@@ -89,6 +89,7 @@ def test___measurement_service___add_configuration__configuration_added(
         ("PinArrayConfiguration", DataType.PinArray1D, ["Pin1", "Pin2"], "test instrument 2"),
     ],
 )
+@pytest.mark.filterwarnings("ignore:.*Pin.*:DeprecationWarning")
```

Comment:
Instead of filtering out warnings, I prefer to use `with pytest.deprecated_call():`. You could mention the deprecated warning in the "then" clause since this is an additional assertion.

https://docs.pytest.org/en/7.1.x/reference/reference.html?highlight=deprecated_call#pytest.deprecated_call

Reply:
That makes sense. I've updated this call to use `with pytest.deprecated_call():`.